core/core_modules/update.js

```diff
@@ -1,128 +1,128 @@
-	});
+        git.getCurrentBranchName(function (err, consoleOutput) {
+                if (err) {
+                        console.debug('Failed to get current branch name\n Error: ' + consoleOutput);
+                        return;
+                }
+                branchName = consoleOutput.trim();
+        });
 
-	// Only allow auto update to be run on the master branch.
-	// By default auto update is enabled but can be turned off in the config.
-	if (branchName === 'master' && (isEnabled || isEnabled === undefined)) {
-		var configUpdatePeriod = this.config.getConfig('update').autoUpdatePeriod;
-		if (configUpdatePeriod) {
-			updatePeriod = configUpdatePeriod;
-		}
-		git.getSHAOfHead(function (err, consoleOutput) {
-			if (err) {
-				console.debug('Failed to get SHA hash of HEAD\n Error: ' + consoleOutput);
-				return;
-			}
-			currentSHA = consoleOutput;
-		});
-		setUpdateTimer();
-	}
+        // Only allow auto update to be run on the master branch.
+        // By default auto update is enabled but can be turned off in the config.
+        if (branchName === 'master' && (isEnabled || isEnabled === undefined)) {
+                var configUpdatePeriod = this.config.getConfig('update').autoUpdatePeriod;
+                if (configUpdatePeriod) {
+                        updatePeriod = configUpdatePeriod;
+                }
+                git.getSHAOfHead(function (err, consoleOutput) {
+                        if (err) {
+                                console.debug('Failed to get SHA hash of HEAD\n Error: ' + consoleOutput);
+                                return;
+                        }
+                        currentSHA = consoleOutput;
+                });
+                setUpdateTimer();
+        }
 };
 
 exports.unload = function() {
-	if (timeout) {
-		clearTimeout(timeout);
-	}
-	shutdown = null;
+        if (timeout) {
+                clearTimeout(timeout);
+        }
+        shutdown = null;
 };
 
 exports.run = function (api, event) {
     api.sendMessage('Updating from git...', event.thread_id);
-	git.pull(function(err, consoleOutput) {
+        git.pull(function(err, consoleOutput) {
         if (err) {
             console.critical(err);
-			api.sendMessage('Update failed. Manual intervention is probably required.', event.thread_id);
-		} else {
-			api.sendMessage('Updating installed NPM packages...', event.thread_id);
-			try {
-				install.update();
-				api.sendMessage('Update successful. Restart to load changes.', event.thread_id);
-			}
-			catch (e) {
-				console.critical(e);
-				api.sendMessage('Update succeeded but NPM package update failed. Manual intervention is probably required.', event.thread_id);
-			}
-		}
-	});
-	return false;
+                        api.sendMessage('Update failed. Manual intervention is probably required.', event.thread_id);
+                } else {
+                        api.sendMessage('Updating installed NPM packages...', event.thread_id);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 space characters but found 24.](https://www.codacy.com/app/theprojectshop/Kassy/file/2555194720/issues/source?bid=3282987&fileBranchId=3291254#l112)